lib/basket.rb

```diff
@@ -0,0 +1,36 @@
+class Basket
```

Comment:
Potential restructure of basket? Move the functionality into order and let basket be a way of storing the dishes. That way you can just keep a write_to method in here and add dishes as they come.